closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -32,15 +32,24 @@
 import jdk.internal.reflect.Reflection;
 import jdk.internal.reflect.CallerSensitive;
 
+import sun.security.action.GetPropertyAction;
+
 public class NativeCrypto {
 
+    private static final String nativeCryptTrace = GetPropertyAction.privilegedGetProperty("jdk.nativeCryptoTrace");
     private static final boolean loaded = AccessController.doPrivileged(
             (PrivilegedAction<Boolean>) () -> {
             Boolean isLoaded = Boolean.FALSE;
             try {
                 System.loadLibrary("jncrypto"); // check for native library
+
+                boolean useCryptoTrace = false;
```

Comment:
Could (works with null value for nativeCryptTrace) :
boolean useCryptoTrace = Boolean.valueOf(nativeCryptTrace);

Reply:
Modified